### PR TITLE
Add PWA Settings to the Store settings

### PR DESCRIPTION
**Pull request by estacioneto**
**Depends on vtex/pwa-graphql#27**

#### What is the purpose of this pull request?

As the title says...

#### What problem is this solving?

No need of the admin-pwa, when the PWA settings can be something in the store.

#### How should this be manually tested?

[Access the workspace](https://pwa--storecomponents.myvtex.com/admin/cms/storefront)

#### Screenshots or example usage

image

image

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/EditorContainer/StoreEditor/Store/PWAForm/components/withPWAMutations.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import React from 'react'
+import { Mutation, MutationFn, MutationResult } from 'react-apollo'
+
+import { handleCornerCases } from '../../utils/utils'
+import UpdateManifest from '../mutations/UpdateManifest.graphql'
+import UpdateManifestIcon from '../mutations/UpdateManifestIcon.graphql'
+import Manifest from '../queries/Manifest.graphql'
+import Styles from '../queries/Styles.graphql'
+
+export interface ManifestMutationData {
```
Comment (brunojdo) [inline react/components/EditorContainer/StoreEditor/Store/PWAForm/components/withPWAMutations.tsx]: I think `ManifestData` is better

Comment (estacioneto) [reply react/components/EditorContainer/StoreEditor/Store/PWAForm/components/withPWAMutations.tsx]: What about [this ManifestData](https://github.com/vtex-apps/admin-pages/pull/166/files#diff-aa7ec14c99aba0a8b5b89c74152d27dbR24)?

File: react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        }),
+      })
+    } finally {
+      setSubmitting(false)
+    }
+  }
+
+  return (
+    <div className="flex flex-column justify-center">
+      <span className="t-heading-5 db">
```
Comment (klzns) [inline react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx]: ⛏  You could use a div instead and remove the `db`

File: react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+        <FormattedMessage id="pages.editor.store.settings.pwa.title" />
+      </span>
+      <div className="pt2">
+        <div className="flex flex-row items-center">
+          <span className="t-heading-7 db w-40">
```
Comment (klzns) [inline react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx]: same

File: react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+                setManifest({ ...manifest, theme_color: color.hex })
+                setColorHistory([...colorHistory, color.hex])
+              }}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="pt2">
+        <div className="flex flex-row items-center">
+          <span className="t-heading-7 db w-40">
```
Comment (klzns) [inline react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx]: same

File: react/components/EditorContainer/StoreEditor/Store/StoreForm/index.tsx [hunk 5]
```diff
@@ -105,7 +113,9 @@ const StoreForm: React.FunctionComponent<Props> = ({ store, intl, mutate }) => {
 
   return (
     <div className="flex flex-column justify-center">
-      <span className="t-heading-5 db">VTEX Store</span>
+      <span className="t-heading-5 db">
```
Comment (klzns) [inline react/components/EditorContainer/StoreEditor/Store/StoreForm/index.tsx]: same

File: messages/context.json [hunk 6]
```diff
@@ -110,6 +110,35 @@
...
+  "pages.editor.store.settings.pwa.orientation.landscape": "Landscape",
+  "pages.editor.store.settings.pwa.orientation.landscape-primary": "Landscape (Primary)",
+  "pages.editor.store.settings.pwa.orientation.landscape-secondary": "Landscape (Secondary)",
+  "pages.editor.store.settings.pwa.orientation.portrait": "Portrait",
+  "pages.editor.store.settings.pwa.orientation.portrait-primary": "Portrait (Primary)",
+  "pages.editor.store.settings.pwa.orientation.portrait-secondary": "Portrait (Secondary)",
+  "pages.editor.store.settings.pwa.update-manifest.success": "Manifest successfuly updated!",
+  "pages.editor.store.settings.pwa.update-manifest.error": "Something wrong occurred while updating the manifest...",
+  "pages.editor.store.settings.pwa.update-manifest-icon.success": "App icon successfuly updated!",
+  "pages.editor.store.settings.pwa.update-manifest-icon.error": "Something wrong occurred while updating the App icon..."
```
Comment (klzns) [inline messages/context.json]: ⛏ in this case we use the character `…` instead.

https://www.w3schools.com/charsets/tryit.asp?deci=8230&ent=hellip

**Pull request merged**